plugins/hls-refactor-plugin/src/Development/IDE/Plugin/CodeAction.hs

```diff
@@ -1740,6 +1815,11 @@ data NotInScope
     | NotInScopeThing T.Text
     deriving Show
 
+mapNotInScope :: (T.Text -> T.Text) -> NotInScope -> NotInScope
+mapNotInScope f (NotInScopeDataConstructor d) = NotInScopeDataConstructor (f d)
+mapNotInScope f (NotInScopeTypeConstructorOrClass d) = NotInScopeTypeConstructorOrClass (f d)
+mapNotInScope f (NotInScopeThing d) = NotInScopeThing (f d)
```

Comment:
Is this function used for anything else? If not, we can likely turn it into a local function (e.g. in the where block) and remove the parameter f with the thing it is supposed to do (adding a ".")

Reply:
Good point! I made it local, changed to more speic name `qualified`, and take qualifier `q` instead of general mapping function `f`.
Addressed in e0eb90a159d0ede5c8b6aab106a79d2d994f1b7f.